docs/design/node-diagram/index.md

```diff
@@ -0,0 +1,13 @@
+# Node diagram
+
+This page depicts the node diagram designs for Autoware Core/Universe architecture.
+
+## Autoware Core
+
+TBD.
+
+## Autoware Universe
+
+![Node diagram](overall-node-diagram-autoware-universe.drawio.svg)
+
+[Open in draw.io for fullscreen](https://app.diagrams.net/?lightbox=1#Uhttps%3A%2F%2Ftier4.github.io%2Fautoware-documentation%2Flatest%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)
```

Comment:
It's better to use `{{ config.site_dir }}`, but we should escape it.
So for now:

```suggestion
[Open in draw.io for fullscreen](https://app.diagrams.net/?lightbox=1#Uhttps%3A%2F%2Fautowarefoundation.github.io%2Fautoware-documentation%2Flatest%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)
```

Reply:
You don't have to commit this now because others cannot view this link yet.

Reply:
Addressed.